docs/how-to-guides/process-image-in-flow.md

```diff
@@ -0,0 +1,58 @@
+# Process image in flow
+PromptFlow defines a contract to represent image data.
+
+## Data class
+`promptflow.contracts.multimedia.Image`
+Image class is a subclass of `bytes`, thus you can access the binary data by directly using the object. It has an extra attribute `source_url` to store the origin url of the image, which would be useful if you want to pass the url instead of content of image to APIs like GPT-4V model.
```

Comment:
don't forget to add this new page to index

Reply:
Done